hw2/04-background.js

```diff
@@ -1 +1,47 @@
 // Add your code here
+let interval;
+const time = document.querySelector('input');
+
+window.onload = (event) => {
+randomColor();  
+};
+
+function changeBackground() {
+  if (!interval) {
+    if (time.value >= 1) {
+      interval = setInterval(randomColor, time.value * 1000);
+    }
+    else {
+      interval = setInterval(randomColor, 0.5 * 1000);
+    }
+  }
+}
```

Comment:
Use named function expressions instead of function declarations as recommended by the [AirBnB style guide](https://github.com/airbnb/javascript#functions--declarations).